Assets/Scripts/Utilities/RenderSettingsManager.cs

```diff
@@ -46,6 +44,15 @@ void Awake()
 
 	void Update()
 	{
+		dayCycleTimer += Time.deltaTime;
+
+		timeOfDay = (-Mathf.Cos(dayCycleTimer/dayCycleTime * Mathf.Rad2Deg) * 0.5f) + 0.5f;
```

Comment:
I was having trouble getting values I expected so I went with something that looked right. I'll go back over this tonight.